inc/parser/xhtml.php

```diff
@@ -147,7 +147,7 @@ function document_end() {
                     }
 
                     // add footnote markup and close this footnote
-                    $this->doc .= $footnote;
+                    $this->doc .= '<span class="content">'.$footnote.'</span>';
```

Comment:
I changed it to a div now.

I have to say it's difficult to restrain myself not improving the semantics here. But that makes more sense after the release (and in a different PR).